main.py

```diff
@@ -0,0 +1,25 @@
+import os
+import urllib.request
+
+import requests
+from dotenv import load_dotenv
+
+if __name__ == '__main__':
+
+    topic = 'cat'
+
+    load_dotenv()
+    API_ACCESS_KEY = os.getenv('API_ACCESS_KEY')
```

Comment:
We use load_dotenv to load env variables to global constants that should be available to the functions no matter if the module is run as a console app or imported. So this can go to the top, below imports, out of `if __name__ == __main__` conditional bock